tsconfig.json

```diff
@@ -30,7 +30,7 @@
     "strictNullChecks": true,
     "strictPropertyInitialization": false,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es5"
+    "target": "es6"
```

Comment:
I think `es6` more reasonable than `es2017`, seems to cover nearly all browsers: https://caniuse.com/es6

![image](https://user-images.githubusercontent.com/4407464/110966748-d61ff280-8312-11eb-92e3-81ffca6709c4.png)